Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -99,26 +136,50 @@ open class PerformanceTestCase: XCTestCase {
         }
         return measures.average
     }
+
+    public func measureMutable(
```

Comment:
I'm not totally sure I understand the differences here, and exactly what is expected to be represented by `closure`. Perhaps the function names could be unified / clarified. Further, perhaps some doc comments could clarify. I'd start with names and types, though.